pandas/tests/generic/test_series.py

```diff
@@ -41,11 +41,16 @@ def test_get_bool_data_preserve_dtype(self):
 
     def test_nonzero_single_element(self):
         # allow single item via bool method
-        ser = Series([True])
-        assert ser.bool()
+        msg = (
+            "NDFrame.bool is now deprecated and will be removed in future releases "
+            "and cases that relied on it will raise in a future version"
+        )
+        with tm.assert_produces_warning(FutureWarning, match=msg):
+            ser = Series([True])
+            assert ser.bool()
```

Comment:
Asserting for warnings should be narrowly scoped - you can remove `ser = Series([True])` from being within this block.

Reply:
done